hack/test.sh

```diff
@@ -24,4 +24,4 @@ mkdir -p "$TEST_RESULTS"
 # shellcheck disable=SC2086
 GODEBUG="tarinsecurepath=0,zipinsecurepath=0" \
     gotestsum --rerun-fails-report=rerunreport.txt --junitfile="$TEST_RESULTS/junit.xml" --format=testname \
-    --rerun-fails="$RERUN_FAILS" --packages="$PACKAGES" -- -cover $TEST_FLAGS "$@"
+    --rerun-fails="$RERUN_FAILS" --packages="${ARGOCD_E2E_PACKAGES:-$PACKAGES}" -- -cover $TEST_FLAGS "$@"
```

Comment:
For non-E2E tests, the variable that controlls this is named `TEST_MODULE` and it is handled in the Makefile itself. Please use the same mechanism, and consider using the same/similar name.